include/nihstro/shader_bytecode.h

```diff
@@ -96,9 +98,8 @@ struct SourceRegister {
             reg.value = index + 0x10;
         else if (type == RegisterType::FloatUniform)
             reg.value = index + 0x20;
-        else {
-            // TODO: Should throw an exception or something.
-        }
+        else
+            reg.value = -1; // Should not happen
```

Comment:
"Should not happen" comments are useless. Actually describe why they cannot happen, and why this way of treating the situation was chosen instead.

Same applies for other instances similar to this one.